iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -915,6 +915,7 @@ def _migrate_surveys(self, v1_index_page, v2_index_page):
                     translated_survey.seo_title = row['seo_title']
                     translated_survey.index_page_description = row['homepage_introduction']
                     translated_survey.index_page_description_line_2 = row['homepage_button_text']
+                    translated_survey.terms_and_conditions = self.map_survey_terms_and_conditions(row)
```

Comment:
Does this work for a translated surveys?
I was checking this one: https://rke-iogt.unicef.io/sections/children-uprooted/french-translation-of-surveys-are-important/
https://rke-iogt.unicef.io/admin/pages/602/edit/
And the terms and conditions don't seem to get migrated?

Reply:
fixed

Reply:
Thanks! Was this related to the unpublished changes field?

Reply:
yes, on the surface it seems it needs revision to work correctly